pipeline/objects/wrappers.py

```diff
@@ -0,0 +1,61 @@
+from pipeline.objects.decorators import pipeline_function, pipeline_model
+from pipeline.objects.pipeline import Pipeline
+from pipeline.objects.variable import PipelineFile, Variable
+
```

Comment:
These imports should be simplified to `from pipeline.objects import Pipeline, PipelineFile, Variable, Graph`